client/webpack.common.config.js

```diff
@@ -1,13 +1,16 @@
 // Common webpack configuration used by webpack.hot.config and webpack.rails.config.
 
+const webpack = require('webpack');
 const path = require('path');
 
 module.exports = {
 
   // the project dir
   context: __dirname,
-  entry: ['jquery', 'jquery-ujs', './assets/javascripts/App'],
-
+  entry: {
+    vendor: ['jquery', 'jquery-ujs'],
```

Comment:
Should other large libraries go here? Such as React, Lodash? or only jquery since it's got to be exported globally?